python/python/glide/async_commands/cluster_commands.py

```diff
@@ -340,3 +341,88 @@ async def time(self, route: Optional[Route] = None) -> TClusterResponse[List[str
             TClusterResponse[List[str]],
             await self._execute_command(RequestType.Time, [], route),
         )
+
+    async def sort(
+        self,
+        key: str,
+        limit: Optional[Limit] = None,
+        order: Optional[OrderBy] = None,
+        alpha: Optional[bool] = None,
+    ) -> List[str]:
+        """
+        Sorts the elements in the list, set, or sorted set at `key` and returns the result.
+        To store the result into a new key, see `sort_store`.
+
+        By default, sorting is numeric, and elements are compared by their value interpreted as double precision floating point numbers.
+
+        See https://valkey.io/commands/sort for more details.
+
+        Args:
+            key (str): The key of the list, set, or sorted set to be sorted.
+            limit (Optional[Limit]): The limit argument for a range query. Defaults to None. See `Limit` class for more information.
+            A tuple specifying the offset and count for limiting the number of results returned.
+                The `limit` parameter takes a tuple `(offset, count)` where `offset` specifies the starting position and `count` specifies the maximum number of elements to return.
+                If `offset` exceeds the length of the returned result, an empty list is returned.
+            order (Optional[OrderBy]): Specifies the order to sort the elements.
+                Can be `OrderBy.ASC` (ascending) or `OrderBy.DESC` (descending).
+            alpha (Optional[bool]): When `True`, sorts elements lexicographically. When `False` (default), sorts elements numerically.
+                Use this when the list, set, or sorted set contains string values that cannot be converted into double precision floating point numbers.
```

Comment:
no need this line.

Reply:
I find it necessary because it needs to be clear that without it - you receive an error

Reply:
So it doesn't make it clear. I didn't understand that it would return an error, i thought you were just trying to give advice.
Change to when x you need to use y otherwise error will occur.
If we don't want to be obliged to the changes in valkey and do not want to say error will, you can change to might occur, or to unexpected behavior.
But as it is now it seems like it's fine to use as you want but you think the user is not too clever so better help him differentiate between numbers and letters.

Reply:
we try to avoid writing - 'you will recive an error' as much as possible because things could change

Maybe we could change that to:

When the list, set, or sorted set contains string values that cannot be converted into double precision floating point numbers, alpha should be set to True